server/game/cards/13.4-BtRK/DragonSkull.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+
+class DragonSkull extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction({ faction: 'targaryen' });
+        this.whileAttached({
+            condition: () => this.areNoDragonsInPlay(),
+            effect: [
+                ability.effects.addKeyword('Intimidate'),
+                ability.effects.addKeyword('Renown')
+            ]
+        });   
+    }
+
+    areNoDragonsInPlay() {
+        return !this.game.getPlayers().some(player => {
+            return player.anyCardsInPlay(card => card.getType() === 'character' && card.hasTrait('Dragon'));

```

Comment:
This works but for future reference, you could also do `!this.game.anyCardsInPlay(card => ...)` instead.